Gemfile

```diff
@@ -0,0 +1,77 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.2.2'
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem 'rails', '~> 7.0.8'
+
+# Rubocop
+gem 'rubocop', '>= 1.0', '< 2.0'
+
+# Use postgresql as the database for Active Record
+gem 'pg'
+
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem 'sprockets-rails'
+
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```

Comment:
- [Optional] Since you're using `pg`, I think you can get rid of this database adapter to improve the efficiency of our app :+1:

Reply:
Thank you for your valuable suggestion. I have removed the sqlite3 database adapter.

Reply:
Fixed.